content/running-amazon-ec2-workloads-at-scale/clone_github.md

```diff
@@ -32,4 +32,29 @@ In order to execute the steps in the workshop, you'll need to clone the workshop
 	    export $output=$(aws cloudformation describe-stacks --stack-name $stack_name --query 'Stacks[].Outputs[?OutputKey==`'$output'`].OutputValue' --output text)
 	    eval "echo $output : \"\$$output\""
 	done
+
 	```
+
+	After executing the above commands you should see the CloudFormation stack outputs listed on your terminal as the following:
+	```bash
+	codeDeployBucket : running-at-scale-codedeploybucket-g1sy53g73ko0
+	fileSystem : fs-38a6dabb
+	instanceProfile : arn:aws:iam::123456789101:instance-profile/running-at-scale-instanceProfile-1DTQQUCMGIU9G
+	eventRule : running-at-scale-eventRule-14TS7W590680Y
+	vpc : vpc-04e37aea04c2de488
+	instanceSecurityGroup : sg-0655496d3a6994056
+	cloud9Environment : cloud9Environment-F48dIcjGd66j
+	dbSecurityGroup : sg-05a351e4895604fd2
+	loadBalancerSecurityGroup : sg-0b88cdbbeffb3823a
+	awsRegionId : us-east-1
+	lambdaFunction : running-at-scale-lambdaFunction-1SR2D6K0S1BU1
+	snsTopic : arn:aws:sns:us-east-1:123456789101:running-at-scale-snsTopic-VOPFFD9IADN7
+	publicSubnet2 : subnet-096d42d2a1b2a2db0
+	dbSubnetGroup : running-at-scale-dbsubnetgroup-t25qeq64q246
+	publicSubnet1 : subnet-0e367e32d2f35cc94
+	codeDeployServiceRole : arn:aws:iam::123456789101:role/running-at-scale-codeDeployServiceRole-AFUU6QGYXOQH
+	```
+
+{{% notice info %}}
+All the configuration files edit automation in this workshop is designed to make your experience smooth and avoid excessive copy-pasting. The commands that will be given use sed to update placeholder values on the file like `%placeholder-value%` with the resource ids created by CloudFormation. However, make sure you detailedly look at each of the configuration files on each step to understand the changes and configurations applied. 
```

Comment:
I agree it can be improved, but suggested text has the word "workshop" multiple times. What about "The automation to edit configuration files in this workshop is designed avoid excessive copy-pasting. The instructions provide commands that use **[sed](https://en.wikipedia.org/wiki/Sed)** to update placeholder values like this `%placeholder-value%` with the resource identifications (ids) created by the output of CloudFormation.  Regardless of the automation you are encouraged to read and understand the configuration files and confirm the changes applied."?